Documentation/PageTsconfig/Templates.rst

```diff
@@ -0,0 +1,128 @@
+:guilabel:`Report` view and another for the :guilabel:`Check link` view. To
+override the :file:`Backend/Report.html` file with a custom template, this
+definition can be added to the :file:`Configuration/page.tsconfig` file of an
+extension:
+
+.. code-block:: typoscript
+   :caption: EXT:site_package/Configuration/page.tsconfig
+
+   # Pattern: templates."composer-name"."something-unique" = "overriding-extension-composer-name":"entry-path"
+   templates.typo3/cms-linkvalidator.1643293191 = my-vendor/my-extension:Resources/Private/TemplateOverrides
+
+If the target extension, identified by its composer name
+`my-vendor/my-extension`, provides the
+:file:`Resources/Private/TemplateOverrides/Templates/Backend/Report.html` file,
+this file is used instead of the default template file from the
+linkvalidator extension.
+
+All core extensions follow the general structure for templates, layouts and
+partials file. If an extension needs to override a partial that
+is located in :file:`Resources/Private/Partials/SomeName/SomePartial.html`, and
+an override has been specified like above to
+:file:`my-vendor/my-extension:Resources/Private/TemplateOverrides`, the system
+looks for the
+:file:`Resources/Private/TemplateOverrides/Partials/SomeName/SomePartial.html`
+file. Similar is the case for layouts.
+
+.. note::
+
+   The path part of the override definition can be set the way an integrator
+   prefers, :file:`Resources/Private/TemplateOverrides` is just an idea here and
+   hopefully not a bad one, further details rely on additional needs. For
+   instance, it is probably a good idea to include the composer or extension
+   name of the source extension into the path (linkvalidator in our example) -
+   or when using overrides based on page or group IDs, to include them in the
+   path.
+
+   The sub-path of the source extension is automatically added by the
+   system when it is searching for override files. If a layout file is located
+   at :file:`Resources/Private/Layouts/ExtraLarge/Main.html` and an override
+   definition uses the :file:`Resources/Private/TemplateOverrides` path, the
+   system will look up
+   :file:`Resources/Private/TemplateOverrides/Layouts/ExtraLarge/Main.html`.
+
+Template overriding is based on the existence of files: Two files are never
+merged. An override definition either takes effect because it actually provides
+a file at the correct position with the correct file name, or it does not and
+the default is used. This can become impractical for large template files. In
+such cases it might be an option to request a split of a large template file
+into smaller partial files so an extension can override a specific partial only.
+
+.. important::
+
+   When multiple override paths are defined and more than one of them contains
+   overrides for a specific template, the override definition with the highest
+   numerical value wins:
+
+   .. code-block:: typoscript
+
```

Comment:
```suggestion
   :caption: EXT:site_package/Configuration/page.tsconfig
```

Reply:
I avoided the caption explicitly, as this case may only happen when two different extensions override a template. It makes no sense to configure both in the same extension or the same config file.

Reply:
If we have no caption it is not clear that we are dealing with Page TSconfig here (by a quick view). How about ` :caption: EXT:another_extension/Configuration/page.tsconfig` ?